cli/src/utilities/runCLI.ts

```diff
@@ -65,6 +65,19 @@ export async function runCLI(toolbox: Toolbox, projectName: string): Promise<Cli
       });
     }
     success(`Great, we'll use ${navigationSelect}!`);
+
+    const useVexo = await prompt.confirm('Would you like to use Vexo for Analytics?', true);
```

Comment:
I think having Vexo as an option flag makes the most sense rather than a prompt question. I am concerned with the CLI having too many questions.

Reply:
We can remove it if necessary. I think there is a benefit of having all third party libraries choices on both ways of initializing your app, the prompt flow works especially well for Vexo since it requires a navigation library to be present.

Reply:
Your concern for the prerequisite of a navigation library is valid. I think that is addressable via an conditional check for whether a navigation package is present. 

My concern re adding Vexo to the core CLI prompts stems largely from CES' stated goal. CES is meant to be a rapid iteration tool rather than as a way to initialize a formal project (hence why we created https://shiprn.dev). Subsequently, the core CLI prompts should only include questions that are necessary for rapid iteration (styling and navigation). I'm even considering removing the auth chunk from CES. I'm happy to add Vexo to ShipRN which is a lot more opinionated.

If you believe there is a case to be made for how analytics are core to rapid iteration, I'm here for it.